apps/web/src/common/modules/modals/NotificationEmailModal.vue

```diff
@@ -3,6 +3,7 @@
         :visible="state.proxyVisible"
         :header-title="$t('COMMON.NOTIFICATION_MODAL.TITLE')"
         class="notification-email-modal-wrapper"
+        :disabled="state.isEditMode ? formState.newNotificationEmail === '' || formState.verificationCode === '' : formState.verificationCode === ''"
```

Comment:
suggestion: I think this would be safer :)

```suggestion
        :disabled="state.isEditMode ? (formState.newNotificationEmail === '' || formState.verificationCode === '') : formState.verificationCode === ''"
```